contents/csharp/IntroductionToCSharp.md

```diff
@@ -0,0 +1,148 @@
+<frontmatter>
+  title: Introduction to C#
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to C#
+
+Author(s:) [Yu Pei, Henry](https://github.com/YuPeiHenry)
+
+## What is C#
+
+C# is an Object-Oriented Programming Language on the .NET framework, and uses Garbage Collection for memory management.
+
+From Wikipedia:
+
+C# (pronounced C sharp) is a general-purpose, multi-paradigm programming language encompassing strong typing, lexically scoped, imperative, declarative, functional, generic, object-oriented (class-based), and component-oriented programming disciplines.
```

Comment:
I am not unsure if the entire definition is needed, maybe you can phrase it your own words such that it easier for the audience to read and understand. 

Maybe you can combine it with 
> C# is an Object-Oriented Programming Language on the .NET framework, and uses Garbage Collection for memory management.
because they are have some overlapping information too.